bps/bps_ApTemplate.yaml

```diff
@@ -10,38 +10,31 @@ campaign: DM-xxxxx
 
 submitPath: ${PWD}/bps/{outCollection}
 computeSite: ncsapool
-# Memory allocated for each quantum, in MBs
+# The default BPS walltime is on the order of 3 days.
```

Comment:
Suggest empty lines before and after this block so it's clearer what the commented text is referring to.